src/applications/health-records/containers/HealthRecordsApp.jsx

```diff
@@ -1,14 +1,16 @@
 import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
+import { Link } from 'react-router';
 
 import DowntimeNotification, { externalServices } from '../../../platform/monitoring/DowntimeNotification';
+import Breadcrumbs from '@department-of-veterans-affairs/formation/Breadcrumbs';
 import Modal from '@department-of-veterans-affairs/formation/Modal';
 import MHVApp from '../../../platform/user/authorization/containers/MHVApp';
 import backendServices from '../../../platform/user/profile/constants/backendServices';
 import RequiredLoginView from '../../../platform/user/authorization/components/RequiredLoginView';
 import { closeModal } from '../actions/modal';
-import Breadcrumbs from '../components/Breadcrumbs';
+// import Breadcrumbs from '../components/Breadcrumbs';
```

Comment:
Similar to above, could you please remove the existing breadcrumb component so that we can review the diff and check how this logic relates to the logic it is replacing?